src/Workspaces/VisualBasic/Portable/Extensions/SyntaxTreeExtensions.vb

```diff
@@ -145,6 +145,27 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
             Return trivia.IsKind(SyntaxKind.SkippedTokensTrivia)
         End Function
 
+        <Extension()>
+        Public Function IsInteractiveOrScript(syntaxTree As SyntaxTree) As Boolean
```

Comment:
I think we got rid of `SourceCodeKind.Script`, so this can probably be `IsInteractive`.